bot/exts/utils/attachment_pastebin_uploader.py

```diff
@@ -0,0 +1,144 @@
+from __future__ import annotations
+
+import re
+
+import aiohttp
+import discord
+from discord.ext import commands
+from pydis_core.utils import paste_service
+
+from bot.bot import Bot
+from bot.constants import Emojis
+from bot.log import get_logger
+
+log = get_logger(__name__)
+
+PASTEBIN_UPLOAD_EMOJI = Emojis.check_mark
+DELETE_PASTE_EMOJI = Emojis.trashcan
+
+
+class EmbedFileHandler(commands.Cog):
```

Comment:
Not sure this class name makes sense.

Reply:
I agree, "file handlers" are a specific thing, this feels more like a MessageAttachmentUploader or something.